gdg/__init__.py

```diff
@@ -144,6 +144,17 @@ def filename_lev(a, f):
     else:
         return levenshtein(a, fn.group(1))
 
+# Returns a list of key-value pairs {image, distance}
+def filter_images_by_lev(name, image_list, max_dist):
+    if max_dist == -1:
+        return map(lambda x: { 'image': x, 'distance': 0 }, image_list)
```

Comment:
`SyntaxError: 'return' with argument inside generator`

Pretty sure you're supposed to yield and then return.